pkg/containerinstall/manager.go

```diff
@@ -0,0 +1,55 @@
+package containerinstall
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"errors"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/env"
+)
+
+type ContainerInstaller interface {
+	Install(ctx context.Context, sub *api.SubscriptionDocument, doc *api.OpenShiftClusterDocument, version *api.OpenShiftVersion) error
+}
+
+type manager struct {
+	conn context.Context
+	log  *logrus.Entry
+	env  env.Interface
+
+	clusterUUID string
+	pullSecret  [2]string
```

Comment:
why do we use an array here? it's seems like a struct or having the username and password as fields inside the manager would make the code easier to understand.

Reply:
cleaned this up